x/poe/keeper/test_common.go

```diff
@@ -222,6 +225,12 @@ func createTestInput(
 	stakingAdapter := stakingadapter.NewStakingAdapter(nil, nil)
 	twasmSubspace := paramsKeeper.Subspace(twasmtypes.DefaultParamspace)
 
+	consensusParamsUpdater := baseapp.NewBaseApp("testApp", log.NewNopLogger(), db, nil)
+	// set the BaseApp's parameter store
+	consensusParamsUpdater.SetParamStore(paramsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramskeeper.ConsensusParamsKeyTable()))
+
+	consensusParamsUpdater.StoreConsensusParams(ctx, app.DefaultConsensusParams)
```

Comment:
`app` is not tgrade/app but wasmd/app. This is fine as we don't want a cyclic dependency but in the `TestValidatorsGovProposal` test you compare against tgrade/app defaults. This causes the test failures.
Please make this an import alias `wasmapp` so that it is clear where it comes from.

Reply:
Thanks! I changed this at the last minute to try and depend on the same values, and broke the tests.